index.html

```diff
@@ -125,17 +127,18 @@ <h3 class="project-name">Tonic</h3>
             <div class="card grid-item card-2">
                 <div class="image-4"></div>
                 <div class="card-info">
-                    <h3 class="project-name">Multi-Post Stories</h3>
+                    <h3 class="project-name">Uber Navigation</h3>
                     <div class="project-info">
-                        <div class="company">CANOPY</div>
+                        <div class="company">Uber</div>
                         <div class="dot"></div>
-                        <div class="position">Back End Dev</div>
+                        <div class="position">Lead Developer</div>
                         <div class="dot"></div>
-                        <div class="year">2015</div>
+                        <div class="year">2018</div>
                     </div>
-                    <p class="desc">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <p class="desc">A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive computer: your car.</p>
                     <ul class="stack">
                         <li class="lang">HTML</li>
+                        <li class="lang">Ruby on Rails</li>
                         <li class="lang">CSS</li>
                         <li class="lang">Javascript</li>
                     </ul>
```

Comment:
Kindly note that for the `About ` section of your project, there are 2 blocks The left one takes 40% and the right one the remaining 60% of the page's width, and the contents of each of them are aligned to the right. Please, implement this for your project so that your project will look closely identical to the design given and it shows that you have attention to details which is a very good skills for a programmer 😄 

Your design :
![image](https://github.com/ayaderaghul/microverse-portfolio/assets/20537712/6733b081-d44a-49bd-bf9d-fcb3e7fdc704)

Figma design :
![image](https://github.com/ayaderaghul/microverse-portfolio/assets/20537712/0bee40af-f06a-41ff-8370-ea816d8f7f11)